cellxgene_schema_cli/cellxgene_schema/write_labels.py

```diff
@@ -218,7 +218,7 @@ def _get_mapping_dict_feature_biotype(self, ids: List[str]) -> Dict[str, str]:
         for i in ids:
             if i.startswith("ERCC"):
                 mapping_dict[i] = "spike-in"
-            elif i.startswith("ENS"):
+            elif i.startswith("ENS") or i.startswith("FB"):
```

Comment:
right now we have to explicitly state which gene prefixes we allow here, so this is another place we'd have to alter when we add new gene types. i'm neutral on keeping this as is, but we could also change this to

```
if i.startswith("ERCC"):
  mapping_dict[i] = "spike-in"
else:
  mapping_dict[i] = "gene"
```

to make it easier for when we add new gene types, if we can safely assume all future gene types will be "genes"

Reply:
I think switching to your suggested change makes sense; it would also be an easy enough change if we do get different feature_biotype requirements in the future